process/block/baseProcess.go

```diff
@@ -692,15 +692,21 @@ func (bp *baseProcessor) removeBlockDataFromPools(headerHandler data.HeaderHandl
 		return process.ErrWrongTypeAssertion
 	}
 
+	timing := make([]time.Time, 0)
```

Comment:
we have something for these types of measurements. Might take a look in core/stopWatch.go. It supports identifiers and is optimized to return the values as []interface{} to match the input for a log print.